codingtest_ajw/4일차/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+    let answer = [];
+   
+    for(let i = 0; i < numbers.length; i++){
+        for(let j = i + 1; j < numbers.length; j++){
+          
+             let inumindex = numbers[i];
+             let jnumindex = numbers[j];
+             answer.push(inumindex + jnumindex);
+        }
+    }
+    answer.sort(function(a, b){
+        return a - b;
+    });
```

Comment:
```suggestion
    answer.sort((a, b) => a - b);
```
k번째수보다 이게 더 먼저 푼 문제인가요? 크게 상관은 없는데 arrow function이 좀 더 깔끔하겠네요